cime_config/testdefs/testmods_dirs/clm/Fates_nat_and_anthro_ignitions/README

```diff
@@ -0,0 +1,26 @@
+Testing capability to read/use lightning frequency & population density
+datasets. Calculates lightning ignitions using the SPITFIRE
+formulation and anthropogenic ignitions using the Li et al. 2012 formulation 
+based on human population density implemented in CTSM-CN's fire model.
+
+Testing does not include a separate test for reading/using a successful
+ignitions dataset which uses the same format and units so handling
+is the same until you're in subroutine area_burnt_intensity where for
+"successful ignitions" the model selects
+currentSite%FDI = 1.0_r8
+cloud_to_ground_strikes = 1.0_r8
+while for lightning the model can use values < 1 for both.
+
+Lara Kueppers (UC-Berkeley) has access to a "successful ignitions"
```

Comment:
This part is confusing. It sounds like this file needs to be added to inputdata and moved in as it's own test.

[ ] Move fates_spitfire_mode = 3 comment to it's own test and move the file into inputdata.

Reply:
This could be done as a separate step from this PR. If the dataset is ready to go and the test can easily be added we might as well. But, if this will take more time an issue should be created and this done later.

Reply:
Created issue #1266 to address this at your suggestion.